public/scripts/indexController.js

```diff
@@ -0,0 +1,9 @@
+(function(module) {
```

Comment:
`indexController.index()` calls `indexView.index()`. Why have two separate files? You could take the functional code from `indexView.index()` and just use it in the `indexController.index()` method.